api/v1beta1/placementapi_webhook.go

```diff
@@ -175,3 +175,25 @@ func ValidateDefaultConfigOverwrite(
 	}
 	return errors
 }
+
+// SetDefaultRouteAnnotations sets HAProxy timeout values of the route
+func (spec *PlacementAPISpecCore) SetDefaultRouteAnnotations(annotations map[string]string) {
+	const haProxyAnno = "haproxy.router.openshift.io/timeout"
+	// Use a custom annotation to flag when the operator has set the default HAProxy timeout
+	// With the annotation func determines when to overwrite existing HAProxy timeout with the APITimeout
+	const placementAnno = "api.placement.openstack.org/timeout"
+	valPlacementAPI, okPlacemenAPI := annotations[placementAnno]
```

Comment:
nit: `okPlacemenAPI `should be `okPlacementAPI`

(Noticed it when I started to replicate this for keystone)